### DISCUSSION TITLE
moving tabs between windows is duplicating them instead of moving them

### DISCUSSION THREAD

**Discussion by icegoddess222** [mbnuqw/sidebery#1752] [General]
hi!

im having this weird issue where when i drag a tab from one window to another, instead of moving that instance of the tab, it will create a new instance of the tab in the new window, while still keeping the original in the previous window. it also does this for multiple selections of tabs.

it doesnt do this when moving the default tabs at the top of firefox, just when im dragging tabs between two sidebery panels. 

this behavior is consistent on my other desktop, both of which are running the latest version of firefox. but my laptop with an older version of firefox behaves normally and drags the tabs to new windows.

so i suspect maybe something was changed in a recent version of firefox that is causing it? i was curious if anyone knows how to fix this issue or if anyone else is also having this issue

i am also unable to move firefox "settings" and "about:" tabs between windows now, but am unsure if that was an issue previously.
it just makes a new tab saying:

"Unavailable URL
Sidebery is unable to open this link due to security-related restrictions of Firefox."

**Comment by jldarling7** [comment]
I am having the same issue. I'm following along to see if there is a fix. Thanks!

**Reply by jldarling7** [reply]
It looks like this has been fixed with the newest Firefox update (Version 129.0.2).

**Reply by icegoddess222** [reply]
i just updated to 129.0.2 and its still doing it for me unfortunately

**Reply by LenianVS** [reply]
It is also doing it for me and its really annoying. Were you able to solve the issue?

**Reply by icegoddess222** [reply]
negative. i also got a new computer and did a complete fresh install of firefox and windows 11 and am still experiencing this bug.

**Comment by clyvari** [comment]
I had the same issue, Firefox 130.0.0 fixed it for me.

See: https://github.com/mbnuqw/sidebery/issues/1723

**Reply by icegoddess222** [reply]
the update to 130.0.0 fixed it for me too!
